sphinx/util/inspect.py

```diff
@@ -91,7 +91,8 @@ def getannotations(obj: Any) -> Mapping[str, Any]:
         __annotations__ = safe_getattr(obj, '__annotations__', None)
     else:
         # Workaround for bugfix not available until python 3.10 as recommended by docs
-        # https://docs.python.org/3.10/howto/annotations.html#accessing-the-annotations-dict-of-an-object-in-python-3-9-and-older
+        # https://docs.python.org/3.10/howto/annotations.html#accessing-the-annotations-dict-of
+        # -an-object-in-python-3-9-and-older
```

Comment:
Possibly unintentional line-break here? (makes the URL more difficult to copy-paste)

Reply:
totally unintended